index.js

```diff
@@ -378,11 +378,27 @@ function Argv (processArgs, cwd) {
   }
 
   var versionOpt = null
-  self.version = function (ver, opt, msg) {
-    versionOpt = opt || 'version'
-    usage.version(ver)
+  self.version = function (opt, msg, ver) {
+    var version = null
+    var versionMsg = null
+
+    if (arguments.length === 1) {
+      version = opt
+      versionOpt = 'version'
+      versionMsg = usage.deferY18nLookup('Show version number')
+    } else if (arguments.length === 2) {
+      versionOpt = opt
+      version = msg
+      versionMsg = usage.deferY18nLookup('Show version number')
+    } else {
```

Comment:
@bcoe I'd like to get the assignment of `versionOpt` into the if/else path too, since `opt || 'version'` wouldn't work if only the `opt` parameter is provided:
```javascript
self.version('4.0.0');

self.version = function (opt, msg, ver) {
  if (arguments.length === 1) {
   version = opt
 } else if (arguments.length === 2) {
   version = msg
 }
 versionOpt = opt || 'version'
}

// version = '4.0.0'
// versionOpt = '4.0.0'
```